src/cmn/team.py

```diff
@@ -237,9 +237,10 @@ def gen_teamsvecs(cls, datapath, output, cfg):
 
             elif 'acceleration' in cfg and 'cuda' in cfg.acceleration:
                 torch = opentf.install_import(cfg.pytorch, 'torch')
-                try: torch.tensor([1.0], device=(device := torch.device(cfg.acceleration if ':' in cfg.acceleration else 'cuda:0')))
```

Comment:
these are new changes. please apply the required changes on the latest version of this file.